portfolio/src/main/webapp/script.js

```diff
@@ -16,13 +16,40 @@
  * Adds a random greeting to the page.
  */
 function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+  //const greetings =
+  //    ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
 
+  const imagesArray = 
+  ['images/youtube_rewind.png','images/chrome_ram.gif', 'images/coffee.png','images/sundar.png'];
   // Pick a random greeting.
```

Comment:
This comment is no longer accurate, it should say "Show the next image" or something like that.